providers/openweathermap.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "openweathermap",
+  "services": [
+    {
+      "id": "default",
+      "baseUrl": "https://api.openweathermap.org"
```

I think it's preferable to include version prefixes into provider's `baseUrl` (or so it's my assumption looking at existing providers), so:

```suggestion
      "baseUrl": "https://api.openweathermap.org/data/2.5"
```
(and update the map's path accordingly)

But maybe the practice has changed since this kinda defies the idea of immutability of providers?

---

That depends on how they version the API - if any update increments the version for all, or if different APIs can be on different version (e.g. /weather and /forecast). I think not including it in the base url is safer.

---

I think @kravchenkoa is correct with his argument here! +1

---

And Ondra already added that to the conventions. Very well!